src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -73,7 +72,7 @@
     private ZipVerifiers() {
     }
 
-    public static Function<ZipStreamWithSignature, ZipInputStream> getPreprocessor(
+    public static ZipPreprocessor getPreprocessor(
         String signatureAlgorithm
```

Comment:
I am not convinced about this change for 2 reasons:
1) It obfuscates a well known Function interface with an unknown one, introducing an extra indirection layer which is unnecessary,  so it reduces clarity even if it reduces verbosity.
2) It's got nothing to do with exception handling :-)

Reply:
Yup ... 
Joshua Bloch agrees with this.
When possible use the standard functional interfaces ...
"Item 3: Favor the use of Standard Functional Interfaces"
https://medium.com/@yury.nino.roa/effective-java-for-lambda-functions-method-references-and-standard-interfaces-6648552f685c